MAUI/Autocomplete/Highlighting-matched-text.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: Highlighting matched text in AutoComplete control | Syncfusion
```

Comment:
check the spelling for the auto complete - Autocomplete

Reply:
Updated changes